### ISSUE TITLE
Resending of order mails from adminhtml does not work, if async mail sending is activated

### ISSUE THREAD

**Issue by shockwavemk** [magento/magento2#4507]
## Steps to reproduce
1. Install Magento from `develop` branch.
2. Create an order
3. Set async email sending "sales_email/general/async_sending" to "true"
4. Go to adminhtml, open created order
5. Click on "Send email"
## Expected result
1. Email is resend to customer
## Actual result
1. Code does not set order to "email_sent" false. Cronjob will not resend order mail.
   
   ``` php
   public function send(Order $order, $forceSyncMode = false)
   {
       $order->setSendEmail(true);
   
       if (!$this->globalConfig->getValue('sales_email/general/async_sending') || $forceSyncMode) {
           if ($this->checkAndSend($order)) {
               $order->setEmailSent(true);
               $this->orderResource->saveAttribute($order, ['send_email', 'email_sent']);
               return true;
           }
       }
       // missing else
       /*
       else {
           $order->setEmailSent(false);
           $this->orderResource->saveAttribute($order, ['send_email', 'email_sent']);
       }
       */
   
       $this->orderResource->saveAttribute($order, 'send_email');
   
       return false;
   }
   ```

**Comment by choukalos** [comment]
Created internal ticket - MAGETWO-52919 to track.

**Comment by KrystynaKabannyk** [comment]
Hello @shockwavemk, this issue has been fixed in the 2.1.0 Release, that's why I'm closing it. If you any questions or additional information regarding the issue feel free to reopen it or create a new one.
